nixos/modules/services/networking/shadowsocks.nix

```
@@ -48,14 +64,27 @@ in
         '';
       };
 
+      package = mkPackageOption pkgs "Shadowsocks" {
+        default = "shadowsocks-libev";
+        example = "services.shadowsocks.package = pkgs.shadowsocks-libev";
+        extraDescription = "Shadowsocks implementation package to use.";
```

Does `mkPackageOption` not accept `description`?
```suggestion
        description = "Shadowsocks implementation package to use.";
```